README.md

```diff
@@ -2,13 +2,13 @@
   <img width="65%" alt="" src="docs/images/logo/fury_github_banner.png"><br>
 </div>
 
-[![Build Status](https://img.shields.io/github/actions/workflow/status/apache/incubator-fury/ci.yml?branch=main&style=for-the-badge&label=GITHUB%20ACTIONS&logo=github)](https://github.com/apache/incubator-fury)
-[![Twitter](https://img.shields.io/twitter/follow/fury_community?logo=twitter&style=for-the-badge)](https://twitter.com/fury_community)
+[![Build Status](https://img.shields.io/github/actions/workflow/status/apache/incubator-fury/ci.yml?branch=main&style=for-the-badge&label=GITHUB%20ACTIONS&logo=github)](https://github.com/apache/incubator-fury/actions/workflows/ci.yml)
+[![Slack Channel](https://img.shields.io/badge/slack-join-3f0e40?logo=slack&style=for-the-badge)](https://join.slack.com/t/fury-project/shared_invite/zt-1u8soj4qc-ieYEu7ciHOqA2mo47llS8A)
```

Comment:
Do we need to use `asf #incubator-fury` channel?

Reply:
Not necessary. Apache Kvrocks now still uses its own slack workspace.

I think it depends on convenience.

Reply:
Got it

Reply:
I don't think so. Let's narrow the channel we maintain. More channels do not mean better community - we just open a channel and no maintainer or even no member in it.

Reply:
DISCLAIMER: Thirdpatry user channel can be always established. The comment above is about "official" channel.

Reply:
OK, let's continue use fury-project channel